apps/decoder_gtfs/test/decoder/gtfs_test.exs

```diff
@@ -16,20 +16,19 @@ defmodule Decoder.GtfsTest do
   end
 
   describe "Decoder" do
-    test "decodes gtfs into stream" do
-      input =
-        File.stream!("VehiclePositions.pb", [], 2048)
-        |> Stream.chunk_every(10)
+    test "decodes gtfs into batch of messages" do
+      decoder = Decoder.Gtfs.new!([])
+      input = File.read!("VehiclePositions.pb")
 
-      output = Decoder.decode(Decoder.Gtfs.new!(chunk_size: 2), input)
+      output = Decoder.decode(decoder, input)
 
       expected =
         File.read!("VehiclePositions.pb")
```

Comment:
just make this `input` as well and avoid double-reading the file.

Reply:
Sorry, I don't understand what you mean here